README.md

```diff
@@ -1 +1,80 @@
-# data-structures
\ No newline at end of file
+# data-structures
+
+UPDATED 12/14/16
+---------- coverage: platform linux2, python 2.7.12-final-0 ----------
+Name                      Stmts   Miss  Cover   Missing
+-------------------------------------------------------
+src/linked_list.py           61      6    90%   36, 47, 66-67, 84-85
+src/stack.py                 14     14     0%   2-28
+src/test_linked_list.py      59      0   100%
+-------------------------------------------------------
+TOTAL                       134     20    85%
+
+
+================================ 24 passed in 0.13 seconds =================================
+py35 inst-nodeps: /media/copenbacon/OS/Users/Conor/codefellows/401/data-structures/.tox/dist/data-structures-0.1.zip
+py35 installed: coverage==4.2,data-structures==0.1,decorator==4.0.10,ipython==5.1.0,ipython-genutils==0.1.0,pexpect==4.2.1,pickleshare==0.7.4,prompt-toolkit==1.0.9,ptyprocess==0.5.1,py==1.4.31,Pygments==2.1.3,pytest==3.0.5,pytest-cov==2.4.0,simplegeneric==0.8.1,six==1.10.0,traitlets==4.3.1,wcwidth==0.1.7
+py35 runtests: PYTHONHASHSEED='1421974174'
+py35 runtests: commands[0] | py.test src --cov=src --cov-report term-missing
+=================================== test session starts ====================================
+platform linux -- Python 3.5.2, pytest-3.0.5, py-1.4.31, pluggy-0.4.0
+rootdir: /media/copenbacon/OS/Users/Conor/codefellows/401/data-structures, inifile: 
+plugins: cov-2.4.0
+collected 24 items 
+
+src/test_linked_list.py ........................
```

Comment:
I don't need the full output text from pytest or tox. I just want the coverage reports.